### PR TITLE
Yet another new litany: Rejection

### PR DESCRIPTION
**Pull request by TheLeopold**
## About The Pull Request
(The ride never ends)
Added new litany. When used, it removes all embedded objects.(Works exactly like rejection mutation)

## Why It's Good For The Game
Gives cruciform an ability to counter carrion spiders, without making it passive buff. Gives cruciform ability to remove foreign objects **because that makes sense** without killing shrapneled disciples in the fight.

## Testing
shot, spidered, threw knifes at disciple
used litany as disciple
all things fell off

## Changelog
:cl:
add: Added new litany: Rejection
/:cl:

### CODE REVIEW

**Comment by Humonitarian** [comment]
Does it work against spider eggs implanted by nurse spiders?

**Comment by TheLeopold** [comment]
wait, we have nurse spiders?
anyway if rejection mutation does, so does this, I literally copypasted the code
Later will try to test if it actually so

**Comment by assortedbeads** [comment]
Good PR
NT needs more prayers that actually do useful shit and making your body spit out sharpnal or crossbow bolts is metal as fuck.

File: code/modules/core_implant/cruciform/rituals/base.dm [hunk 1]
```diff
@@ -61,6 +61,34 @@
 	set_personal_cooldown(H)
 	return TRUE
 
+/datum/ritual/cruciform/base/reject
+	name = "Rejection"
+	phrase = "Ad tollendum zizania a corpore"
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/base.dm:66]:
```suggestion
	phrase = "Ad tollendum zizania a corpore."
```

**Review by hyperioo — APPROVED** [review]

**Comment by hyperioo** [comment]
Might need a future PR to balance it depending on user feedback

**Pull request merged**